post-install.txt

```diff
@@ -12,13 +12,17 @@ chroot /rootfs /usr/bin/git clone -b master https://github.com/openhab/openhabia
 if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 ln -s /opt/openhabian/openhabian-setup.sh /rootfs/usr/local/bin/openhabian-config
 
-echo "[openHABian] === Start executing 'openhabian-setup.sh' ==="
+echo -n "[openHABian] Backup installation configuration..."
+cp /rootfs/boot/installer-config.txt /rootfs/opt/openhabian/installer-config.txt.netinstalled
```

Comment:
This needs to be done in your function above, otherwise an existing system will be without config.